My Series List/My Series List/View/MainView/MainViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MainViewController.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 18/11/2021.
+//
+
+import UIKit
+
+class MainViewController: UIViewController, MainViewDelegate {
```

Comment:
Rename as MainController or ListController. If you choose to use List as the name of this screen, please fix on other classes